src/modules/tests/test-data/test-data.selector.ts

```diff
@@ -153,8 +153,12 @@ export const hasLegalRequirementBeenCompleted = (
 
 export const getVehicleChecks = (state: CatBUniqueTypes.TestData): CatBUniqueTypes.VehicleChecks => state.vehicleChecks;
 
-export const getTellMeQuestion = (state: CatBUniqueTypes.VehicleChecks): VehicleChecksQuestion =>
-  tellMeQuestions.find(question => question.code === get(state, 'tellMeQuestion.code'));
+export const getTellMeQuestion = (state: CatBUniqueTypes.VehicleChecks): VehicleChecksQuestion => {
```

Comment:
This test data selector is tightly coupled to CatB.  Is there a plan to refactor this for CatBE (duplication of selectors and renaming of file or category specific selectors within this file)

Reply:
It will have to be refactored and/or duplicated and should be part of the Show Me/Tell Me question tickets coming up next sprint.

Because of the difference, I think we will most likely end up with a Cat B and a Cat BE version of the selector.